xamarin/Xamarin.Forms
# Fixed the rounding issue when the stepper value is incremented.

When the increment value is small like say 0.5 per step. This is due to Double being used as the value so when it's unboxed from the binding object precision is lost.

### Description of Change ###
In the Stepper bindable property for the value this is a Double so when the value is unboxed from the Object it's not 100% precise such that when the stepper is used in the platform renderer of say Android 0.5 - 0.5 doesn't result in zero but some slight non-zero value.



### Issues Resolved ### 
- fixes #5168 

### API Changes ###
None

### Platforms Affected ### 


- Core/XAML (all platforms)
- iOS
- Android
- UWP

### Behavioral/Visual Changes ###


To fix the issue Math.Round has been used to a precision of 6 decimal places so if a used was to use the stepper with a precision greater than this they will face an issue.  But lets be honest they are then using the wrong tool for the job...

### Before/After Screenshots ### 


Not applicable

### Testing Procedure ###
Use the test case g5168 on your chosen platform to confirm that the error has been fixed.

### PR Checklist ###
